pkg/controller/controller_binding.go

```diff
@@ -695,19 +750,26 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 		}
 
 		if toUpdate.Status.OrphanMitigationInProgress {
-			s := fmt.Sprintf(`Orphan mitigation successful for ServiceBinding "%v/%v"`, binding.Namespace, binding.Name)
+			s := fmt.Sprintf(
+				`%s "%s/%s": Orphan mitigation successful`,
+				typez, binding.Namespace, binding.Name,
+			)
 			c.setServiceBindingCondition(toUpdate,
 				v1beta1.ServiceBindingConditionReady,
 				v1beta1.ConditionFalse,
 				successOrphanMitigationReason,
 				s)
 		} else {
+			s := fmt.Sprintf(
```

Comment:
I don't think we need the context for which binding this is, since this is just going onto a condition attached to the resource.